src/main/resources/hudson/plugins/active_directory/Messages.properties

```diff
@@ -13,3 +13,4 @@ TlsConfiguration.TrustAllCertificates = (Insecure) Trust all Certificates
 TlsConfiguration.JdkTrustStore = JDK TrustStore
 
 TlsConfiguration.AdministrativeMonitor.DisplayName = Active Directory TLS Configuration Monitor
+TlsConfiguration.WarningMessage = Disabling TLS in FIPS mode is not allowed. Either enable StartTls or Require TLS.
```

Comment:
No need to mention `LEGACY_FORCE_LDAPS_PROPERTY` right? (I'm OK with that, I'm just asking for confirmation)

Reply:
I did not mention `LEGACY_FORCE_LDAPS_PROPERTY` as the user controls only checkboxes while configuring.

Reply:
LEGACY_FORCE_LDAPS_PROPERTY is, as it's named, legacy. It was the way of configuring TLS before the UI included the checks. For me, the text should only mention the UI